extensions/wikia/AdEngine3/src/ad-context.js

```diff
@@ -360,6 +360,16 @@ export default {
 					}
 				}
 			},
+			vmg: {
+				enabled: false,
+				slots: {
+					TOP_LEADERBOARD: {
+						sizes: [
+							[3, 3]
```

Comment:
Just asking,  it's `2x2` on mobile and `3x3` on desktop?

Reply:
Yes, like UAP sizes :+1: